src/cmd_all/src/playground.rs

```diff
@@ -224,22 +224,28 @@ pub async fn playground() -> Result<()> {
                     .join("start-service.sh");
                 if cmd_path.exists() {
                     tracing::info!("start connector-node with prefix_bin {}", prefix_bin);
-                    let mut cmd = Command::new(cmd_path);
-                    cmd.arg("-p").arg("50051");
-                    cmd.stdout(std::process::Stdio::piped());
-                    let mut child = cmd.spawn().expect("failed to start connector node");
-                    let stdout = child.stdout.take().expect("failed to open stdout");
-                    let _child_handle = tokio::spawn(async move { child.wait().await });
-                    let _stdout_handle = tokio::spawn(async move {
-                        let mut reader = BufReader::new(stdout).lines();
-                        while let Some(line) =
-                            reader.next_line().await.expect("failed to read line")
-                        {
-                            eprintln!("{}", line);
+                    let mut child = Command::new(cmd_path)
+                        .arg("-p")
+                        .arg("50051")
+                        .stderr(std::process::Stdio::piped())
+                        .spawn()?;
+                    let stderr = child.stderr.take().unwrap();
+
+                    let _child_handle = tokio::spawn(async move {
+                        signal::ctrl_c().await.unwrap();
+                        let _ = child.start_kill();
+                    });
```

Comment:
I think it is ok to kill in an explicit way

Reply:
> I feel this is unnecessary because the child process is already killed currently

Yes. But I do have encountered situations where the connector node not gets killed. 🤔 Not sure what's happening under the hood.